modules/modal/react/stories/stories_VisualTesting.tsx

```diff
@@ -17,12 +18,33 @@ export default withSnapshotsEnabled({
 
 const noop = () => {}; // eslint-disable-line no-empty-function
 
+const TestContent = () => {
+  const content = (
+    <div
+      style={{
+        color: 'white',
+        zIndex: 1,
+        position: 'relative',
+      }}
+    >
+      This should be invisible if the Modal is rendering correctly, because the Modal should use the
+      PopupStack which will set a zIndex and place the modal and its overlay _on top_ of the this
+      text. If the text renders on top of the overlay, the text will show up
```

Comment:
Remove `"this"` (or `"the"`) in `"on top of the this text"`. Also, `"If the text renders on top of the overlay, the text will show up"` reads a bit odd to me (wasn't sure if it was one of your half-finished thoughts 😆 ). I wonder if `"Otherwise, the text will be visible on top of the overlay."` would read better, or you could just remove the second sentence entirely since it's already implied.